js/script.js

```diff
@@ -0,0 +1,201 @@
+// ------------------------
+// LOGIC
+// ------------------------
+
+const Board = (() => {
+  let cells = new Array(9).fill(" ");
+
+  const fillCell = (cellNumber, cellSymbol) => {
+    cells[cellNumber] = cellSymbol;
+  }
+
+  const isCellTaken = (cellNumber) => {
+    return cells[cellNumber] != " ";
+  }
+
+  // const resetBoard = () => {
+  //   cells = new Array(9).fill(" ");
+  // }
+
+  return {cells, fillCell, isCellTaken}
+
+})();
+
+const Player = function(name, symbol) {
```

Comment:
shouldn't the player be responsible for the play? thats like the only thing a player does: make moves. then we can keep all the moves done by a player here as well can can use that to check win if we wish to go that way.

Reply:
@SunnyAsar I agree that player makes moves but I think it should be handled by board as it is that resource that the player is changing.
What are the differences/advantages of making Player handle move vs Board?